frontend/components/product-list/sections/FilterableProductsSection/RefinementMenu.tsx

```diff
@@ -90,12 +91,18 @@ const MenuItem = React.memo(function RefinementListItem({
       includedAttributes: [attribute],
    });
    const { createURL } = useCurrentRefinements();
-   const href = createURL({
-      attribute,
-      type: 'disjunctive',
-      value,
-      label,
-   });
+   const url = new URL(
+      createURL({
+         attribute,
+         type: 'disjunctive',
+         value,
+         label,
+      })
+   );
+   // The url created by InstantSearch doesn't have the correct item type slug.
+   const path = url.pathname.split('/').filter((part) => part !== '');
+   const itemTypeHandle = encodeDeviceItemType(value);
+   const href = `${url.origin}/${path[0]}/${path[1]}/${itemTypeHandle}${url.search}`;
```

Comment:
We only want to build it this way if "value" is the itemType, which means this assumes that the only RefinementMenu is the ItemType one. Might be okay for now, but I thought I saw another pull changing that too.

Reply:
Yea, we'll handle that when we start reusing this component for different facet types